Runtime/Core/AmazonGameLiftPlugin.Core.csproj

```diff
@@ -26,6 +26,9 @@
   </ItemGroup>
   <ItemGroup>
     <!-- See: https://stackoverflow.com/questions/15241889/i-didnt-find-zipfile-class-in-the-system-io-compression-namespace -->
+    <Reference Include="GameLiftServerSDK">
```

Comment:
Is this meant to be in this PR or PR 100?

Reply:
Removed